pkgs/by-name/fr/freesweep/package.nix

```diff
@@ -3,31 +3,41 @@
   ncurses,
   lib,
   stdenv,
-  updateAutotoolsGnuConfigScriptsHook,
+  autoconf,
+  automake,
+  pkg-config,
   installShellFiles,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "freesweep";
-  version = "1.0.2";
+  version = "1.0.2-unstable-2024-04-19";
 
   src = fetchFromGitHub {
     owner = "rwestlund";
     repo = "freesweep";
-    rev = "v${version}";
-    hash = "sha256-iuu81yHbNrjdPsimBrPK58PJ0d8i3ySM7rFUG/d8NJM";
+    rev = "68c0ee5b29d1087d216d95875a7036713cd25fc0";
+    hash = "sha256-ZnAH7mIuBMFLdrtJOY8PzNbxv+GDEFAgyEtWCpTH2Us=";
   };
 
+  patches = [
+    ./0001-include-strings.h.patch
+    ./0002-fix-Wformat-security.patch
+    ./0003-remove-ac_func_malloc.patch
```

Comment:
Add a short description why each patch is necessary, and _if possible_, link to an upstream tracking issue or commit